<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaRon</class>
 <widget class="QMainWindow" name="ventanaRon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background: qlineargradient(
        spread:pad, x1:0, y1:0, x2:1, y2:0,
        stop:0 rgba(255, 220, 150, 255), /* Color claro, hacia la izquierda */
        stop:1 rgba(230, 134, 60, 255)  /* Color naranja oscuro, hacia la derecha */
    );

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="rTitulo">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>1011</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-family: &quot;Calibri&quot;;         /* Fuente Calibri */
    font-size: 30pt;                /* Tamaño de 20pt */
    color: #AC1F01;                 /* Color rojo oscuro especificado */
    font-weight: bold;            /* Normal, ajusta a bold si necesitas */
    text-align: left;               /* Alineación izquierda, cámbialo a center si es necesario */
	background: none;

}
</string>
    </property>
    <property name="text">
     <string>Opciones de Ron disponibles… Seleccione su favorito</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rBoton">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>850</y>
      <width>220</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #e67e22;
    color: white;                /* Texto blanco */
    font-family: Calibri;         /* Fuente Calibri */
    font-size: 22px;              /* Tamaño de la fuente */
    font-weight: bold;  
    border-radius: 20px;          /* Bordes redondeados */
    padding: 10px 20px;           /* Espaciado interno para agrandar el botón */
}

QPushButton:hover {
   background-color: #d35400;     /* Color de fondo al pasar el mouse */
}</string>
    </property>
    <property name="text">
     <string>Volver</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rTropicalPunch">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>150</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ bebidasRon/tropicalPunch.png);
background: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="rRonPunch">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>500</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ bebidasRon/ronPunch.png);
background: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="rPinaColada">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>150</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ bebidasRon/pinaColada.png);
background: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="rPomeloRumSplash">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>150</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ bebidasRon/pomeloRumSplash.png);
background: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="rRonTonic">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>500</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ bebidasRon/ronTonicTropical.png);
background: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="rTropicalRumStorm">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>500</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ bebidasRon/tropicalRumStorm.png);
background: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>rPomeloRumSplash</zorder>
   <zorder>rTitulo</zorder>
   <zorder>rBoton</zorder>
   <zorder>rTropicalPunch</zorder>
   <zorder>rRonPunch</zorder>
   <zorder>rPinaColada</zorder>
   <zorder>rRonTonic</zorder>
   <zorder>rTropicalRumStorm</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../Bebidas/bebidasRon/bebidasRon.qrc"/>
 </resources>
 <connections/>
</ui>
